### PR TITLE
clean up salt.utils for using 2018.3.3

### PR DESCRIPTION
**Pull request by gtmanfred**

### CODE REVIEW

File: _modules/pip.py [hunk 1]
```diff
@@ -15,8 +15,8 @@
 import pkg_resources
 
 # Import salt libs
-import salt.utils
-from salt.utils import namespaced_function
+from salt.utils.platform
```
Comment (rallytime) [inline _modules/pip.py]: `from` --> `import` :)

Comment (gtmanfred) [reply _modules/pip.py]: bah, yes

**Comment by rallytime** [comment]
Fixes https://github.com/saltstack/salt-jenkins/issues/903

**Comment by rallytime** [comment]
@gtmanfred Looks like some of the tests are unhappy here.

**Comment by gtmanfred** [comment]
yup, looks like the locale stuff is broken in 2018.3.3 for opensuse and debian/ubuntu

**Comment by gtmanfred** [comment]
yup, one of them is going to have to be fixed when we switch over to using the branchs to bootstrap instead of stable releases.

https://github.com/saltstack/salt/pull/50259

**Comment by rallytime** [comment]
@gtmanfred Hrm, the ubuntu/debian tests are still having a rough time with this one. Can you take another look?

**Comment by gtmanfred** [comment]
Yeah, i fixed that already, i force pushed over top, but did not clean the old tests https://travis-ci.org/saltstack/salt-jenkins/builds/447904951 this is the correct build for this change.

**Pull request merged**